src/main/java/codechicken/translocators/init/ModContent.java

```diff
@@ -39,13 +45,17 @@
     @ObjectHolder ("item_translocator")
     public static ContainerType<ContainerItemTranslocator> containerItemTranslocator;
 
+    public static Tag<Item> diamondNuggetTag = new ItemTags.Wrapper(new ResourceLocation("forge:nuggets/diamond"));
```

Comment:
I wasn't sure if I was supposed to put this in the `ModContent` class, or the `Translocators` class.

Reply:
I believe there is a bug with ObjectHolder, this may crash at runtime due to that, I would move it to the Translocators main class for now and we can address it later.